### ISSUE TITLE
Attempting to fetch value instead of handling error Failed precondition: Could not load dynamic library 'libhip_hcc.so'

### ISSUE THREAD

**Issue by vuquocan1987** [ROCmSoftwarePlatform/tensorflow-upstream#1022]
**System information**

- Ubuntu 20.04
- TensorFlow installed from (source or binary): pip install tensorflow-rocm
- TensorFlow version (use command below): 2.2.0
- Python version: 3.7
- GPU: rx 580
- ROCM 3.5.1:

**Describe the current behavior**
When I invoke code:
`
model = tf.keras.models.Sequential([
tf.keras.layers.Flatten(input_shape=(28, 28)),
tf.keras.layers.Dense(128, activation='relu'),
tf.keras.layers.Dropout(0.2),
tf.keras.layers.Dense(10)
])
`
Or anything relate to a tensorflow operation I got error:

`Attempting to fetch value instead of handling error Failed precondition: Could not load dynamic library 'libhip_hcc.so'`
**Describe the expected behavior**

Additional Info
I have setup the tensorflow-rocm with the following commands :

sudo apt install libnuma-dev
wget -q -O - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | sudo apt-key add -
echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' | sudo tee /etc/apt/sources.list.d/rocm.list
sudo apt install rocm-dkms
sudo usermod -a -G video $LOGNAME
echo 'export PATH=$PATH:/opt/rocm/bin:/opt/rocm/profiler/bin:/opt/rocm/opencl/bin' | sudo tee -a /etc/profile.d/rocm.sh
sudo apt install rocm-libs miopen-hip rccl
pip3 install tensorflow-rocm

**Comment by papadako** [comment]
I had the same problem. What I did to resolve it was to copy `/opt/rocm/hip/lib/libamdhip64.so.3.5.30501` to` /usr/lib/libhip_hcc.so` and run ldconfig. I am not sure though why (it seems like a packaging error).

**Comment by vuquocan1987** [comment]
> I had the same problem. What I did was to copy `/opt/rocm/hip/lib/libamdhip64.so.3.5.30501` to` /usr/lib/libhip_hcc.so` and run ldconfig. I am not sure though why (it seems like a packaging error).

Did you solved the problem by doing so :-?

**Comment by papadako** [comment]
> > I had the same problem. What I did was to copy `/opt/rocm/hip/lib/libamdhip64.so.3.5.30501` to` /usr/lib/libhip_hcc.so` and run ldconfig. I am not sure though why (it seems like a packaging error).
> 
> Did you solved the problem by doing so :-?

Yep

**Comment by vuquocan1987** [comment]
I am sorry if this seem obvious, but what is idconfig? where should i run it?

**Comment by vuquocan1987** [comment]
> > > I had the same problem. What I did was to copy `/opt/rocm/hip/lib/libamdhip64.so.3.5.30501` to` /usr/lib/libhip_hcc.so` and run ldconfig. I am not sure though why (it seems like a packaging error).
> > 
> > 
> > Did you solved the problem by doing so :-?
> 
> Yep

I am sorry if this seem obvious: 
1. So you copy  `/opt/rocm/hip/lib/libamdhip64.so.3.5.30501` to` /usr/lib/libhip_hcc.so` mean, you copy and rename the file right?
2  What is idconfig? where should i run it?

**Comment by papadako** [comment]
As root you should:
a) `cp /opt/rocm/hip/lib/libamdhip64.so.3.5.30501 /usr/lib/libhip_hcc.so`
b) `ldconfig` (the first letter is L - this is used for updating the links/cache of shared libraries, you can run it from anywhere)

**Comment by vuquocan1987** [comment]
Thank you very much. That solved my problem!

**Comment by jnairfamily** [comment]
> As root you should:
> a) `cp /opt/rocm/hip/lib/libamdhip64.so.3.5.30501 /usr/lib/libhip_hcc.so`
> b) `ldconfig` (the first letter is L - this is used for updating the links/cache of shared libraries, you can run it from anywhere)

Thank you. This solved the problem
